tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -662,19 +627,15 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 		// Arrange.
 		const wpAdmin = new WpAdminPage( page, testInfo );
 		await setup( wpAdmin );
```

Comment:
What about this section? which activates and deactivates the experience for each test.

Reply:
Seems like a separate PR to me - this is an anti-pattern I see in a lot of tests